manifests/image-references

```diff
@@ -0,0 +1,32 @@
+kind: ImageStream
+apiVersion: image.openshift.io/v1
```

Comment:
Not strictly needed but it would be nice (and consistent) to start all yamls files with:

```
---
apiVersion:
```